tst/EnergyPlus/unit/ReportSizingManager.unit.cc

```diff
@@ -124,3 +140,169 @@ TEST( ReportSizingManager, GetCoilDesFlowT )
 	DataSizing::CalcSysSizing.deallocate();
 	
 }
+TEST( ReportSizingManager, RequestSizingSystem ) {
+	ShowMessage( "Begin Test: ReportSizingManager, RequestSizingSystem" );
+
+	int write_stat;
+	// Open the Initialization Output File
+	OutputFileInits = GetNewUnitNumber();
+	{ IOFlags flags; flags.ACTION( "write" ); flags.STATUS( "UNKNOWN" ); gio::open( OutputFileInits, "eplusout.eio", flags ); write_stat = flags.ios(); }
+
+	std::string CompName; // component name
+	std::string CompType; // component type
+	std::string SizingString; // input field sizing description
+	int SizingType; // integer type of sizing requested
+	Real64 SizingResult; // autosized value of coil input field
+	bool PrintWarning; // true when sizing information is reported in the eio file
+	std::string CallingRoutine; // calling routine
+
+	DataHVACGlobals::CoolingCapacitySizing;
```

Comment:
@Nigusse Never mind, these lines are just surplus.  SizingType is set to this elsewhere.